README.md

```diff
@@ -11,6 +11,18 @@ construction of fixtures via decorators.
 
 ## Usage
 
+pytest_terraform provides a `terraform` decorator with the following parameters:
+
+| Argument             | Required? | Type    | Default      | Description |
+| -----                | :---:     | ---     | ---          | ---         |
+| `terraform_dir`      | yes       | String  |              | Terraform module (directory) to execute. |
+| `scope`              | no        | String  | `"function"` | [Pytest scope](https://docs.pytest.org/en/stable/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) - should be one of: `function`, `class`, `module`, `package`, or `session`. |
```

Comment:
so i haven't really tested class, module, package.. i'm not sure it works with the xdist combo unless we handle them as sessions, technically our implementation should be fine with them across different processes since we're tracking fixture deps.

Reply:
I'll make sure leave a note that `function` and `session` should be targeted and `class`, `module`, `package` may not work as expected with xdist